test/metabase/models/session_test.clj

```diff
@@ -16,7 +16,10 @@
     (k/insert Session
               (k/values [{:id         "the-greatest-day-ever"
                           :user_id    user-id
-                          :created_at (metabase.util/->Timestamp "1980-10-19")}
+                          :created_at (metabase.util/->Timestamp "1980-10-19T05:05:05.000Z")}
+                         {:id         "even-more-greatness"
+                          :user_id    user-id
+                          :created_at (metabase.util/->Timestamp "1980-10-19T05:08:05.000Z")}
```

Comment:
you shouldn't do this, refer to a namespace like `metabase.util` without `:require`-ing it. It might not be loaded yet so this makes the code implicitly dependent on being ran after some other namespace loaded `metabase.util`.